tutorials/preprocessing/plot_70_fnirs_processing.py

```diff
@@ -0,0 +1,199 @@
+# and the unwanted heart rate component has been removed, we can extract epochs
+# related to each of experimental conditions.
+#
+# First we extract the events of interest and visualise them to ensure they are
+# correct.
+
+events, _ = mne.events_from_annotations(raw_fnirs, event_id={'1.0': 1,
+                                                             '2.0': 2,
+                                                             '3.0': 3})
+event_dict = {'Control': 1, 'Tapping/Left': 2, 'Tapping/Right': 3}
+fig = mne.viz.plot_events(events, event_id=event_dict,
+                          sfreq=raw_fnirs.info['sfreq'])
+fig.subplots_adjust(right=0.7)  # make room for the legend
+
+
+###############################################################################
+# Next we define the range of our epochs, the rejection criteria,
+# baseline correction, and extract the epochs. We visualise the log of which
+# epochs were dropped.
+
+reject_criteria = dict(hbo=80e-6)
+tmin, tmax = -5, 15
+
+epochs = mne.Epochs(raw_fnirs, events, event_id=event_dict,
+                    tmin=tmin, tmax=tmax,
+                    reject=reject_criteria, reject_by_annotation=True,
+                    proj=True, baseline=(None, 0), preload=True,
+                    detrend=None, verbose=True)
+epochs.plot_drop_log()
+
+
+###############################################################################
+# View consistency of responses across trials
+# -------------------------------------------
+#
+# Now we can view the haemodynamic response for our tapping condition.
+# We visualise the response for both the oxy- and deoxyhaemoglobin, and
+# observe the expected peak in HbO at around 6 seconds consistently across
+# trials, and the consistent dip in HbR that is slightly delayed relative to
+# the HbO peak.
+
+epochs['Tapping'].plot_image(combine='mean', vmin=-30, vmax=30)
+
+
+###############################################################################
+# We can also view the epoched data for the control condition and observe
+# that it does not show the expected morphology.
+
+epochs['Control'].plot_image(combine='mean', vmin=-30, vmax=30)
+
+
+###############################################################################
+# View consistency of responses across channels
+# ---------------------------------------------
+#
+# Simillarly we can view how consistent the response was across the optode
+# pairs that we selected. All the channels in this data we located over the
+# motor cortex, and all channels show a similar pattern in the data.
+
+fig, (cont, tapping) = plt.subplots(nrows=2, ncols=2, figsize=(15, 6))
+clims = dict(hbo=[-20, 20], hbr=[-20, 20])
+epochs['Control'].average().plot_image(axes=[cont[0], tapping[0]],
+                                       show=False, clim=clims)
+epochs['Tapping'].average().plot_image(axes=[cont[1], tapping[1]],
+                                       show=False, clim=clims)
+fig.show()
```

Comment:
this plot is really great. Only thing missing is text at the top of each column saying "Control" and "Tapping". Here's also a suggestion for streamlining (avoiding `cont` and `tapping` variables) that is untested but I think should work:

```
fig, axes = plt.subplots(nrows=2, ncols=2, figsize=(15, 6))
clims = dict(hbo=[-20, 20], hbr=[-20, 20])
epochs['Control'].average().plot_image(axes=axes[:, 0], clim=clims)
epochs['Tapping'].average().plot_image(axes=axes[:, 1], clim=clims)
```

Is the `show=False` followed by `fig.show()` necessary to get this to work?